lib/libutils/isoc/bget.c

```diff
@@ -868,10 +873,10 @@ void brel(buf, poolset)
 	assert(poolset->totalloc >= 0);
 	poolset->numdrel++;	       /* Number of direct releases */
 #endif /* BufStats */
-#ifdef FreeWipe
-	V memset_unchecked((char *) buf, 0x55,
-			   (MemSize) (bdh->tsize - sizeof(struct bdhead)));
-#endif /* FreeWipe */
+	if (wipe)
```

Comment:
Minor: rest of the source file uses `{ }` even for single line conditioned blocks. Suggest to add braces here and line 964.

Reply:
OK